keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -179,10 +178,27 @@ or expect preloaded images since boot.
 
 ## Design Details
 
-Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image. The hash map
-will not be persisted to disk, in alpha. For alpha explicitly, we will not reuse or add other state manager concepts to kubelet.
+Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image. It has been decided that the hash map will be persisted to disk, in alpha.
```

Comment:
can we remove "in memory" then, if the map will be persisted to disk?

Reply:
will be both in memory and persisted

Reply:
should be resolved